features/polyExplorer/src/components/clusterStories/MatrixBubblesChart.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useRef } from "react";
+import { PolyChart } from "@polypoly-eu/poly-look";
+import * as d3 from "d3";
+
+import "./MatrixBubblesChart.css";
+
+const MatrixBubblesChart = ({ data, bubbleColor, textColor, strokeColor }) => {
+    const matrixBubbleChartWidth = 65;
+    const matrixBubbleChartHeight = 50;
+    const chartRef = useRef();
+
+    useEffect(() => {
+        d3.select(chartRef.current);
```

Comment:
This doesn't seem to do anything? I get the impression neither this `useEffect` block nor `chartRef` is needed, unless I'm missing something.

Reply:
You are completely right about it, I'll delete it.